jbpm/jbpm-flow-builder/src/main/java/org/jbpm/compiler/canonical/ProcessToExecModelGenerator.java

```diff
@@ -201,6 +205,25 @@ public void accept(CompilationUnit cu) {
         }
     }
 
+    private static class AddProcessAnnotation implements Consumer<CompilationUnit> {
+
+        private final String processId;
+
+        public AddProcessAnnotation(String processId) {
+            this.processId = processId;
+        }
+
+        @Override
+        public void accept(CompilationUnit cu) {
+            Optional<ClassOrInterfaceDeclaration> clazz = cu.findFirst(ClassOrInterfaceDeclaration.class);
```

Comment:
This could save us a few lines. I leave to your consideration which way you prefer.
```suggestion
            ClassOrInterfaceDeclaration clazz = cu.findFirst(ClassOrInterfaceDeclaration.class).orElseThrow(()->new NoSuchElementException("Cannot find class declaration in the template"));
```